test/upgrade/main.go

```diff
@@ -114,7 +126,12 @@ func ConfigTestSetup(ctx context.Context) []*configTest {
 		ct := configTest{}
 		ct.agonesVersion = agonesVersion
 		if agonesVersion == "DEV" {
-			ct.agonesVersion = DEV
+			ct.agonesVersion = Dev
+			// Game server container cannot be created at DEV version due to go.mod only able to access
```

Comment:
The https://github.com/googleforgames/agones/blob/main/test/sdk/go/sdk-client-test.go would be at Dev-1 version, but the rest of Agones (controller, allocator, etc.) would be at Dev version. So it doesn't test for Dev changes to `sdk-client-test.go`. I _think_ that should be fine since `sdk-client-test.go` itself is tested separately, and what we're looking for in this test is cross-version compatibility. WDYT?

Reply:
Makes sense!